.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Release
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
+
+permissions:
+  contents: write
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v4
+
+      - name: Set up environment 🧰
+        run: |
+          echo "TAG_NAME=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+          echo "VERSION=${GITHUB_REF#refs/tags/v}" >> $GITHUB_ENV
+
+      - name: Build 🔨
+        run: |
+          make build-linux
+          mv ./build/milkywayd ./build/milkywayd-${VERSION}-linux-amd64
+          SHA256_SUM=$(sha256sum ./build/milkywayd-${VERSION}-linux-amd64 | awk '{ print $1 }')
+          echo "SHA256_SUM=${SHA256_SUM}" >> $GITHUB_ENV
+
+      - name: Generate release notes 📜
+        run: |
+          echo "## Features" > release_notes.txt
+          echo "" >> release_notes.txt
+          echo "## Bug Fixes" >> release_notes.txt
+          echo "" >> release_notes.txt
```

Comment:
Let's remove this, we can add them manually since the sections are not always the same (some time there are only features, other only bug fixed, other only dependecy updated and so on)